frontend/src/components/ForgotPassword/styles.ts

```diff
@@ -0,0 +1,45 @@
+import { Box, TextField } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+export const RegistrationFormWrapper = styled(Box)`
+  height: 82vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const BorderForm = styled(RegistrationFormWrapper)`
+  border: 2px solid black;
+  height: 72vh;
+  border-radius: 20px;
+  width: 500px;
+`;
+
+export const StyledSpan = styled(Box)`
+  margin-top: 20px;
+  text-align: end;
+`;
+
+export const StyledSpanOr = styled(Box)`
+  display: block;
+`;
+
+export const StyledSpanEnd = styled(StyledSpan)`
+  margin-top: 15px;
+`;
+
+export const TextFieldWrap = styled(TextField)`
```

Comment:
this blocks label to appear in all 3 forms. TextField comp is an input not a simple container

now its:
<textfield>
    <textfield />
</textfield>

should be:
<div>
    <textfield />
</div>

Reply:
done